apps/gov-website/pages/proposals/new.tsx

```diff
@@ -161,7 +182,7 @@ const useFormSubmit = () => {
 
 				await api.tx.governance
 					.submitProposal(
-						proposalData.get("proposalExtrinsic"),
+						proposalCall ? getProposalExtrinsic(api, proposalCall) : undefined,
```

Comment:
so does this work, by passing it the entire extrinsic?

Reply:
I haven't had any luck testing with the `scheduler`